pkg/network/ebpf/c/protocols/postgres/decoding.h

```diff
@@ -211,8 +244,14 @@ int socket__postgres_process(struct __sk_buff* skb) {
         bpf_tail_call_compat(skb, &protocols_progs, PROG_POSTGRES_PROCESS_PARSE_MESSAGE);
         return 0;
     }
+    postgres_kernel_msg_count_t* pg_msg_counts = get_pg_msg_counts_map(pkt);
+    // TODO: condition below increases complexity by 6K instructions
+    if (!pg_msg_counts) {
+        log_debug("POSTGRES kernel telemetry, socket map not found.");
+        return 0;
+    }
```

Comment:
1. can be moved inside `postgres_entrypoint`
2. Currently the implementation does not allow us to decode the traffic, so "no telemetry == no monitoring". In the case we couldn't find the telemetry map - we should not report telemetry.

Reply:
Right, it should be moved to postgres_entrypoint. This was my attempt to reduce complexity placing this non-null condition in different places, but it does not help. I need condition to check that pointer is not null, but this condition increases complexity.

Reply:
done